src/backends/mod.rs

```diff
@@ -54,6 +55,12 @@ pub fn execute_command<'a>(
                 }
             }
         }
+        CommandSpec::Clean => {
+            let output_root = &config.apps[0].output_root;
+            if let Err(e) = remove_dir_all(&output_root.display().to_string()) {
+                error!("lingo was unable to delete build folder! {e}");
```

Comment:
More or less the same error message is produced by `remove_dir_all`

Reply:
yes `remove_dir_all` produces and error and here we handle it.

Reply:
But it seems like the error is: "lingo was unable to delete build folder! Could not delete folder: ..." which seems a little redundant